src/Shouldly.Tests/Strings/ShouldNotMatch.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.Strings
+{
+    public class ShouldNotMatch : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Cheese".ShouldNotMatch(@"\w+", "Some additional context");
+        }
+
+        protected override string ChuckedAWobblyErrorMessage
+        {
+            get
+            {
+                return "\"Cheese\" should not match \"\\w+\" but was \"Cheese\" " +
```

Comment:
Hey, sorry for the delay @bradarv90 

an example of the message generators is https://github.com/shouldly/shouldly/blob/master/src/Shouldly/MessageGenerators/ShouldBeMessageGenerator.cs

You could create a ShouldMatchMessageGenerator and wire it up in the same way, that then lets you customise the messages